{
	"dataType": "CVE_RECORD",
	"dataVersion": "5.0",
	"cveMetadata": {
		"cveId": "CVE-2020-1984",
		"assignerOrgId": "d6c1279f-00f6-4ef7-9217-f89ffe703ec0",
		"assignerShortName": "palo_alto",
		"state": "PUBLISHED",
		"datePublished": "2020-04-08T00:00:00",
		"dateUpdated": "2020-04-08T00:00:00.000000Z",
		"dateReserved": "2020-04-08T00:00:00"
	},
	"containers": {
		"cna": {
			"providerMetadata": {
				"orgID": "d6c1279f-00f6-4ef7-9217-f89ffe703ec0",
				"shortName": "palo_alto",
				"dateUpdated": "2020-04-08T18:41:58"
			},
			"title": "Secdo: Privilege escalation via hardcoded script path",
			"descriptions": [
				{
					"lang": "en",
					"value": "Secdo tries to execute a script at a hardcoded path if present, which allows a local authenticated user with 'create folders or append data' access to the root of the OS disk (C:\\) to gain system privileges if the path does not already exist or is writable. This issue affects all versions of Secdo for Windows.",
					"supportingMedia": [
						{
							"base64": "false",
							"type": "text/html",
							"value": "<pre>Secdo tries to execute a script at a hardcoded path if present, which allows a local authenticated user with 'create folders or append data' access to the root of the OS disk (C:\\) to gain system privileges if the path does not already exist or is writable. This issue affects all versions of Secdo for Windows.</pre>"
						}
					]
				}
			],
			"affected": [
				{
					"vendor": "Secdo",
					"product": "Secdo",
					"platforms": [
						"Windows"
					],
					"versions": [
						{
							"status": "affected",
							"version": "all versions"
						}
					]
				}
			],
			"problemTypes": [
				{
					"descriptions": [
						{
							"type": "CWE",
							"lang": "en",
							"description": "CWE-73 External Control of File Name or Path",
							"cweId": "CWE-73"
						}
					]
				}
			],
			"metrics": [
				{
					"cvssV3_1": {
						"version": "3.1",
						"vectorString": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:H/I:H/A:H",
						"attackVector": "LOCAL",
						"attackComplexity": "LOW",
						"privilegesRequired": "LOW",
						"userInteraction": "NONE",
						"scope": "UNCHANGED",
						"confidentialityImpact": "HIGH",
						"integrityImpact": "HIGH",
						"availabilityImpact": "HIGH",
						"baseScore": 7.8,
						"baseSeverity": "HIGH"
					}
				}
			],
			"workarounds": [
				{
					"lang": "en",
					"value": "This issue can be mitigated by :\n- Ensure unprivileged users do not have 'create folder' access on the root of filesystem such as C:\\.\nor\n- Creating a folder named C:\\Common and ensuring unprivileged users do not have 'create folder' access.\n",
					"supportingMedia": {
						"base64": false,
						"type": "text/html",
						"value": "This issue can be mitigated by :\n- Ensure unprivileged users do not have 'create folder' access on the root of filesystem such as C:\\.\nor\n- Creating a folder named C:\\Common and ensuring unprivileged users do not have 'create folder' access.\n"
					}
				}
			],
			"solutions": [
				{
					"lang": "en",
					"value": "This product is no longer supported and the issue is not going to be fixed. The issue can be completely mitigated by ensuring that unprivileged users do not have 'create folder' access on the root of the filesystem such as C:\\ or on a folder named C:\\Common.",
					"supportingMedia": {
						"base64": false,
						"type": "text/html",
						"value": "This product is no longer supported and the issue is not going to be fixed. The issue can be completely mitigated by ensuring that unprivileged users do not have 'create folder' access on the root of the filesystem such as C:\\ or on a folder named C:\\Common."
					}
				}
			],
			"references": [
				{
					"tags": [
						"x_refsource_CONFIRM"
					],
					"url": "https://security.paloaltonetworks.com/CVE-2020-1984"
				}
			],
			"timeline": [
				{
					"time": "2020-04-08T16:00:00.000Z",
					"lang": "eng",
					"value": "Initial publication"
				}
			],
			"credits": [
				{
					"lang": "en",
					"value": "We like to thank Eviatar Gerzi of CyberArk Labs Team for discovering and reporting this issue."
				}
			],
			"source": {
				"defect": [
					"PDV-1613"
				],
				"discovery": "EXTERNAL"
			},
			"datePublic": "2020-04-08T16:00:00.000Z",
			"x_generator": {
				"engine": "Vulnogram 0.0.9"
			},
			"x_affectedList": [
				"Secdo"
			]
		}
	}
}
